### PR TITLE
Feature/pdf test interface

### PR DESCRIPTION
**Pull request by tofixx**

### CODE REVIEW

File: views/pdf.pug [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+                    div.form-group
+                        label.control-label.col-sm-4(for='file')= 'image'
+                        div.col-sm-8
+                            input#file.form-control(type='text', name='file', placeholder='give me any url to an image', required)
+
+                    div.form-group
+                        label.control-label.col-sm-4(for='quality', name='quality')= 'quality'
+                        div.col-sm-8
+                            select#quality.form-control(name='quality')
+                                option(value='')= 'printer (default)'
```
Comment (franticBIT) [inline views/pdf.pug]: seems unnecessary to have this twice

File: views/pdf.pug [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    h1= 'pdf test inteface'
+    p= 'give me an pdf url and let me compress it quickly for you'
+
+    div.container-fluid
+        div.row
+            div.col-sm-6
+                form#configure-form.form-horizontal(method="get" action="/pdf")
+
+                    div.form-group
+                        label.control-label.col-sm-4(for='file')= 'image'
```
Comment (franticBIT) [inline views/pdf.pug]: hmm... those copy & paste errors. 😀

File: views/pdf.pug [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+                        div.col-sm-8
+                            input#file.form-control(type='text', name='file', placeholder='give me any url to an image', required)
```
Comment (franticBIT) [inline views/pdf.pug]: another one

File: views/pdf.pug [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+                            select#quality.form-control(name='quality')
+                                option(value='')= 'printer (default)'
+                                option(value='screen')= 'screen'
+                                option(value='printer')= 'printer'
+
+                    div.form-group
+                        div.col-sm-offset-4.col-sm-8
+                            button#update-pdf-button.btn.btn-primary(type='submit')= 'do the magic...'
+            div.col-sm-6
+                div#result-pdf.image
```
Comment (franticBIT) [inline views/pdf.pug]: could be taller... maybe `100vh`?

File: routes/pdf.js [hunk 5]
```diff
@@ -3,6 +3,7 @@ const router = new express.Router();
 const config = require('config');
 const pdfLoader = require('../middleware/pdfLoader');
 const checkQueryParams = require('../middleware/checkQueryParams');
+const onlyDevelopment = require('../middleware/onlyDevelopment');
```
Comment (franticBIT) [inline routes/pdf.js]: We could also just not at those routes, if in production env.

File: views/pdf.pug [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+    script(src='//maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js', integrity='sha384-Tc5IQib027qvyjSMfHjOMaLkfuWVxZxUPnCJA7l2mCWNIpG9mGCD8wGNIcPD7Txa', crossorigin='anonymous')
+
+    link(rel='stylesheet', href='//code.jquery.com/ui/1.12.1/themes/base/jquery-ui.css')
+    script(src='//code.jquery.com/ui/1.12.1/jquery-ui.js')
+
+    script(src='/js/test.js')
+
+
+block content
+    h1= 'pdf test inteface'
```
Comment (franticBIT) [inline views/pdf.pug]: interface

**Comment by franticBIT** [comment]
What about a test that calls the page?

File: views/pdf.pug (hunk 1 above)
Comment (tofixx) [inline views/pdf.pug]: done

File: views/pdf.pug (hunk 4 above)
Comment (tofixx) [inline views/pdf.pug]: done

File: routes/pdf.js (hunk 5 above)
Comment (tofixx) [inline routes/pdf.js]: maybe :)

**Pull request merged**